pypit/armlsd.py

```diff
@@ -122,13 +125,15 @@ def ARMLSD(fitsdict, reuseMaster=False, reloadMaster=True):
             ###############
             # Generate an array that provides the physical pixel locations on the detector
             slf.GetPixelLocations(det)
-            # Determine the edges of the spectrum (spatial)
+            # Determine the edges of the spectra (spatial)
             if ('trace'+settings.argflag['reduce']['masters']['setup'] not in settings.argflag['reduce']['masters']['loaded']):
                 ###############
                 # Determine the edges of the spectrum (spatial)
                 lordloc, rordloc, extord = artrace.trace_slits(slf, slf._mstrace[det-1], det, pcadesc="PCA trace of the slit edges")
                 slf.SetFrame(slf._lordloc, lordloc, det)
                 slf.SetFrame(slf._rordloc, rordloc, det)
+                # Initialize maskslit
+                slf._maskslits[det-1] = np.array([False] * slf._lordloc[det-1].shape[1])
```

Comment:
I think this is slightly quicker:
`slf._maskslits[det-1] = np.zeros(slf._lordloc[det-1].shape[1], dtype=np.bool)`

Reply:
FWIW, `np.bool` and `bool` are exactly the same.  I used to the same thing (like setting `dtype=np.float` instead of `dtype=float`), but I only do this now if I want to use a numpy specific type, like `np.bool_`.  In a quick search, I ran across [this issue](https://github.com/numba/numba/issues/1311) that may be useful to keep in mind if we start to use numba more regularly.

Reply:
edited